contractcourt/channel_arbitrator.go

```diff
@@ -12,15 +12,6 @@ import (
 	"github.com/lightningnetwork/lnd/lnwire"
 )
 
-const (
-	// broadcastRedeemMultiplier is the additional factor that we'll scale
```

Comment:
Hmm, OK, I buy the idea that broadcasting for outgoing HTLCs is less urgent than for incoming since you have the CLTV delta on the other side to make sure you don't lose money. Can I flip it so the broadcast delta becomes 20 and it gets halved in the timeout case instead of getting doubled in the incoming case? So that BroadcastDelta can be taken as a safe upper bound on the broadcast delta in all cases.

Reply:
Isn't this race condition why we need to keep watching for remote spends even after we publish our sweep transaction? (which is implemented in the new sweeper)